invenio_app_ils/circulation/loaders/schemas/json/loan_request.py

```diff
@@ -104,4 +105,12 @@ def postload_checks(self, data, **kwargs):
                     "request_expire_date": [message],
                 }
             )
+        elif end - start < loan_request_offset:
+            message = "The request end date can only be {} days after the request start date.".format(loan_request_offset.days)
```

Comment:
Question: Is this error message correct? If I understand correctly it should be something like:
```suggestion
            message = "The requested start date must be at least {} days from today.".format(loan_request_offset.days)

```
""""
Or am I missing something?

Reply:
This is indeed more straightforward, will update this, thank you!